### ISSUE TITLE
Video Player widget

### ISSUE THREAD

**Issue by cityjumper** [mathworks/widgets-toolbox#41]
Would be nice ot have a Video Player Widget, instead of having to create a uihtml (which does not work out of the box with mp4), and play around with the events to have a control slider bar, and the speed play, etc...

**Comment by rjackey** [comment]
I agree. I'll keep this in mind and also put in a suggestion for the app designer team.  If you build one yourself, consider contributing it to MATLAB File Exchange.
